nuxeo-runtime/nuxeo-stream/src/test/java/org/nuxeo/lib/stream/tests/TestLibChronicle.java

```diff
@@ -53,9 +54,18 @@
 public class TestLibChronicle implements StoreFileListener {
     protected static final Log log = LogFactory.getLog(TestLibChronicle.class);
 
+    protected final static String OS = System.getProperty("os.name").toLowerCase();
+
+    public final static boolean IS_WIN = OS.startsWith("win");
+
     @Rule
     public TemporaryFolder folder = new TemporaryFolder();
 
+    @Before
+    public void skipWindowsThatDontCleanTempFolder() {
+        org.junit.Assume.assumeFalse(IS_WIN);
```

Comment:
You can use `ConditionalIgnoreRule.IgnoreWindows` on the class, I don't check if you can depend on `nuxeo-runtime-test`. Otherwise, you can use `SystemUtils.IS_OS_WINDOWS` from commons-lang3.

Reply:
Furthermore, you can import static `assumeFalse`.

Reply:
Noted, but this is a pure java lib and I don't want to add a dep that don't reduce the number of lines. fixing import